apps/src/music/views/Callouts.tsx

```diff
@@ -72,55 +79,93 @@ const Callouts: React.FunctionComponent = () => {
     (state: {music: MusicState}) => state.music.currentCallout
   );
 
-  const availableCallout = callout.id
-    ? availableCallouts[callout.id]
-    : undefined;
+  const [isMoving, setIsMoving] = useState(false);
+
+  const calloutIds = callout?.id?.split('---');
+
+  const validCallouts: AvailableCallout[] | undefined = calloutIds?.map(
+    calloutId => availableCallouts[calloutId]
+  );
 
   const calloutIndex = callout.index;
 
+  interface Target {
+    left: number;
+    top: number;
+  }
+
+  const targets: Target[] = [];
+
+  validCallouts?.map(validCallout => {
```

Comment:
super nit: should this be a `forEach` since we're not really mapping anything? alternatively, could keep the map and create the `targets` array inline? i.e.
```
const targets = validCallouts?.map(validCallout => ... return { left: ..., top: ...}).filter(target => !!target);
```

Reply:
Thanks for catching this.  Updated to `forEach`.